src/screens/HomeScreen/HomeScreen.tsx

```diff
@@ -1,51 +1,98 @@
-import React, {useState} from 'react';
-import {Text, View, TouchableOpacity} from 'react-native';
+import React, {useContext, useState} from 'react';
+import {Text, View, TouchableOpacity, Alert} from 'react-native';
+import {AuthContext} from '../../context/AuthContext';
 import withHeader from '../../helpers/withHeader';
+import {DataStoreHook} from '../../hooks/dataStoreHook';
 import Strings from '../../i18n/en';
+import {updateStatus} from '../AuthScreen/Util';
 import {HomeViewStyle} from './styles';
 
 const HomeScreen = () => {
-  const [status, SetStatus] = useState<string>('active');
-  const [oooStatus, SetOOOStatus] = useState<boolean>(false);
-  const changeStatus = () =>
-    status === 'active' ? SetStatus('idle') : SetStatus('active');
-  const changeoooStatus = () =>
-    oooStatus === false ? SetOOOStatus(true) : SetOOOStatus(false);
-  return (
-    <View style={HomeViewStyle.container}>
-      {oooStatus === false && (
-        <View>
+  const [loader, setLoader] = useState<boolean>(false);
+
+  const {loggedInUserData, setLoggedInUserData} = useContext(AuthContext);
+
+  const changeStatus = (status:string) => {
+    
+    setLoader(true)
+    loggedInUserData &&
+      updateStatus(status).then(() => {
+        setLoggedInUserData({
+          ...loggedInUserData,
+          status: status
+        }) 
+        DataStoreHook(
+          'userData',
+          JSON.stringify({
+            ...loggedInUserData,
+            status: status
+
+          }),
+        );
+      }).catch((err) => Alert.alert(err)).finally(() => setLoader(false))
+  };
+
+    const renderScreen = () => {
+      if(loggedInUserData?.status === "ooo" ){
```

Comment:
we can have constants for `ooo` and `active`. It would be better for future references.

Reply:
will resolve in next commit .